src/ResourceManager/Sql/Commands.Sql/ServerCommunicationLink/Cmdlet/GetAzureSqlServerCommunicationLink.cs

```diff
@@ -0,0 +1,76 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Sql.ServerCommunicationLink.Model;
+
+namespace Microsoft.Azure.Commands.Sql.ServerCommunicationLink.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmSqlServerCommunicationLink", 
+        ConfirmImpact = ConfirmImpact.None)]
+    public class GetAzureSqlServerCommunicationLink : AzureSqlServerCommunicationLinkCmdletBase
```

Comment:
@adumitr What is the OutputType for this cmdlet?  Please add the attribute for OutputType with the cmdlet OutputType